widgetry/src/widgets/mod.rs

```diff
@@ -630,7 +630,7 @@ impl Widget {
                 w.restore(ctx, prev);
             }
         } else if self.widget.can_restore() {
-            if let Some(ref other) = prev.top_level.find(self.id.as_ref().unwrap()) {
+            if let Some(ref other) = prev.maybe_find(self.id.as_ref().unwrap()) {
```

Comment:
I wanted to make top_level private so I could confidently mess with it within Panel, this was the only external usage.